src/dropdown/Dropdown.js

```diff
@@ -102,9 +102,18 @@ export default class Dropdown extends React.Component {
     return onChange(value);
   }
 
+  onInputKeyDown = (e) => {
```

Comment:
does it event make sense to have a dropdown with an empty list of options? wouldn't it be better to improve `@props` and forbid `props.options` to be empty?